src/Og.php

```diff
@@ -58,21 +58,21 @@ public static function createField($field_name, $entity_type, $bundle) {
     }
 
     // Add the field to the form display manager.
-    $displayForm = EntityFormDisplay::load($entity_type . '.' . $bundle . '.default');
-    if (!$displayForm->getComponent($field_name) && $widgetDefinition = $og_field->widgetDefinition()) {
-      $displayForm->setComponent($field_name, $widgetDefinition);
-      $displayForm->save();
-    }
-
-    // Define the view mode for the field.
-    if ($fieldViewModes = $og_field->viewModesDefinition()) {
-      $prefix = $entity_type . '.' . $bundle . '.';
-      $viewModes = \Drupal::entityManager()->getStorage('entity_view_display')->loadMultiple(array_keys($fieldViewModes));
-
-      foreach ($viewModes as $key => $viewMode) {
-        $viewMode->setComponent($field_name, $fieldViewModes[$prefix . $key])->save();
-      }
-    }
+//    $displayForm = EntityFormDisplay::load($entity_type . '.' . $bundle . '.default');
```

Comment:
For now it's comment out since this api got removed. Once the tests are passing I'll fix this one.